packages/edge-config/src/utils/tracing.ts

```diff
@@ -1,12 +1,23 @@
-import {
-  trace as traceApi,
-  type Tracer,
-  type Attributes,
-} from '@opentelemetry/api';
+import type { TraceAPI, Tracer, Attributes } from '@opentelemetry/api';
 import { name as pkgName, version } from '../../package.json';
 
+// Use a symbol to avoid having global variable that is scoped to this file,
+// as it can lead to issues with cjs and mjs being used at the same time.
+const edgeConfigTraceSymbol = Symbol.for('@vercel/edge-config:global-trace');
+
+/**
+ * Allows setting the `@opentelemetry/api` tracer to generate traces
+ * for Edge Config related operations.
+ */
+export function setTracer(tracer: TraceAPI): void {
```

Comment:
Seems like the recommended pattern is to use `TracerProvider`?

https://opentelemetry.io/docs/concepts/instrumentation/libraries/#getting-a-tracer